src/pages/Main/PopStoreSection/PopStore/PopStore.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+
+import './PopStore.scss';
+
+class PopStore extends Component {
+  render() {
+    const { popStore } = this.props;
+
+    return (
+      <>
+        {popStore.map(e => (
```

Comment:
```suggestion
        {popStore.map(store => (
```
e(이벤트 이외의 경우), el, data 같은 추상적인 이름보다 코드만 보고서도 내용을 추론할 수 있도록 구체적인 이름으로 정해주세요

Reply:
넵!